EKF/control.cpp

```diff
@@ -61,18 +61,18 @@ void Ekf::controlFusionModes()
 
 			// send alignment status message to the console
 			if (_control_status.flags.baro_hgt) {
-				ECL_INFO("EKF aligned, (pressure height, IMU buf: %i, OBS buf: %i)", (int)_imu_buffer_length, (int)_obs_buffer_length);
+				ECL_INFO("%llu: EKF aligned, (pressure height, IMU buf: %i, OBS buf: %i)", (unsigned long long)_imu_sample_delayed.time_us, (int)_imu_buffer_length, (int)_obs_buffer_length);
```

Comment:
Isn't the ECL_INFO_TIMESTAMPED implemented for this exact case?

Reply:
Yes, but it is only working for statements without additional arguments. Otherwise I would added it already to this line.

Reply:
You can add VA_ARGS to the macro and it will then work with arguments.

Reply:
Yes, but the amount of argument values (also zero) can vary and not all the compiler running in CI support that. I tried hard.